README.md

```diff
@@ -1,24 +1,178 @@
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Greeting page 
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ install ruby
+ ruby --version
+ install sqlite3
+ sqlite3 --version
+ gem install rails 
+ rails --version
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/eudondian/hello-rails-react.git
+```
+### Install
+
+Install this project with:
+
+```sh
+  cd hello-rails-r-app
+  bundle install
+ ```
+ ```sh
+ raiils db:create
+ ```
+ ```sh
+  raiils db:create
+ ```
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ./bin/dev
+
+  http://localhost:5000/api/greetings
+
+  ```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Esther Udondian**
+
+- GitHub: [@Click here](https://github.com/eudondian)
+- Twitter: [@Click here](https://twitter.com/EUdondian)
+- LinkedIn: [@Click here](https://www.linkedin.com/in/esther-udondian/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  Add style.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/eudondian/hello-rails-react/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project give me a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for the learning experinece,  Gregoire Vella on Behance for the design, and my partners for all help given
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] Please fix the MIT License link for a professional README file and repo. 🙏